### ISSUE TITLE
The example on the README failed to run on Ubuntu 20.04

### ISSUE THREAD

**Issue by gotoxu** [tokio-rs/io-uring#88]
OS: Ubuntu 20.04.2 LTS
`/proc/version`: Linux version 5.4.0-70-generic (buildd@lcy01-amd64-004) (gcc version 9.3.0 (Ubuntu 9.3.0-17ubuntu1~20.04)) #78-Ubuntu SMP Fri Mar 19 13:29:52 UTC 2021

Cargo.toml: 
```toml
[package]
name = "io-uring-rs"
version = "0.1.0"
edition = "2018"

# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html

[dependencies]
io-uring = "0.5"
```

Program running results :
```shell
    Finished dev [unoptimized + debuginfo] target(s) in 0.29s
     Running `target/debug/io-uring-rs`
thread 'main' panicked at 'read error: -22', src/main.rs:28:5
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

Is this `io-uring` bug? Or is it a problem with the program itself? Please help to answer, thanks.

**Comment by quininer** [comment]
duplicate https://github.com/tokio-rs/io-uring/issues/87

**Comment by gotoxu** [comment]
Closed due to duplicate issues #87
